### DISCUSSION TITLE
Can I work on more than one project at a time? Do I need to install Reach in every project I work on?

### DISCUSSION THREAD

**Discussion by TheChronicMonster** [reach-sh/reach-lang#1169] [General]
This question is about directory (folder) organization. The tl;dr answer is that you can work on as many projects as you'd like. Methodical directory architecture will help you stay organized.

You should feel free to structure your directories any way you'd like. However, this is our suggestion:

* We suggest the `reach` install live in a folder called `Reach` that is installed on the top level.
* Project folders should be contained inside subdirectories to the `Reach` folder. As such, any project build in the path of this Reach folder has access to the Reach language repository

Example:
home>Reach (execute curl install command here)
home>Reach>tut (.rsh, .js, .mjs files live here)
home>Reach>project1 (more .rsh, .js files)
home>Reach>project2 (more .rsh, .js files)

This organization allows a single install of Reach in the top-level directory and straightforward access from all sub-directories

Here is a screenshot of several projects in a Reach directory:

![directories](https://user-images.githubusercontent.com/9493272/174585888-a06409a6-5020-455c-8aa2-3a223b80057d.png)

If the `ls` command doesn't show the reach folder, it is likely one level up. In that case, instead of executing `./reach run`, use `../reach run`.

**Comment by TheChronicMonster** [comment]
Keep in mind that this is our suggestion, but you should use a pattern that is meaningful and easy for you to use. It's your workflow!

**Comment by ThangBK2009** [comment]
You can export reach to global Environment, then you can use for all project in anywhere directory.
